packages/shell-api/src/collection.ts

```diff
@@ -85,6 +85,14 @@ type CollStatsShellOptions = CollStatsOptions & {
   indexDetailsName?: string;
 };
 
+function optionalFirstString(arg1?: string|Document, arg2?: Document): [arg1: null|string, arg2: null|Document] {
```

Comment:
Is this gross? I can't find any helpers where we have the first param optional. ie. the `function(name, options)` OR `function(options)` pattern. The driver does it extensively, but our linting rules really don't like the way they do it.

Reply:
Also is there a better place to put this kind of helper or is it OK here until we need it elsewhere?

Reply:
https://github.com/mongodb-js/mongosh/blob/9b3d27adac4d425d2b892b79e00e0988c2b11fb2/packages/shell-api/src/shard.ts#L116-L119 is the typical way to do this in JS, and I’d also inline this into the API methods for this case too